applications/luci-app-zerotier/htdocs/luci-static/resources/view/zerotier/interface.js

```diff
@@ -0,0 +1,119 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2022 ImmortalWrt.org
+ */
+
+'use strict';
+'require fs';
+'require ui';
+'require view';
+
+return view.extend({
+	load: function() {
+		return fs.exec('/sbin/ifconfig').then(function(res) {
```

Comment:
Please revise to an rpcd or ucode script. We definitely should not be parsing `ifconfig` output to display info here.

A ucode script is better suited to this. See an [example](https://github.com/openwrt/luci/pull/6456#issuecomment-1655109643) here. Might this be the reason for the `jsonfilter` dep?

Reply:
Just some quick notes
https://ucode.mein.io/module-fs.html#popen
https://ucode.mein.io/module-core.html#json
https://ucode.mein.io/module-core.html#call
https://man7.org/linux/man-pages/man8/ip.8.html
ip --json
https://man7.org/linux/man-pages/man8/ifconfig.8.html